modules/admin_manual/pages/installation/nginx_configuration.adoc

```diff
@@ -203,58 +203,6 @@ location = /data/\.ocdata {
 }
 ----
 
-[[prevent-access-log-entries-when-accessing-thumbnails]]
-=== Prevent access log entries when accessing thumbnails
-
-When using eg. the Gallery App, any access to a thumbnail of a picture
```

Comment:
UNDO the DELETE !!!
We just have to extend/replace gallery by the new app name (mediaviewer I guess) !!
This part is important